src/components/TransactionDetails/TransactionDetailsList.jsx

```diff
@@ -1,32 +1,12 @@
-/* eslint-disable camelcase */
 import React from 'react';
+import TransactionDetailsListItem from './TransactionDetailsListItem.jsx';
```

Comment:
could you add empty line between 3d party libraries from package.json and your own code imports please?